services/website/src/routes/index.svelte

```diff
@@ -1,4 +1,5 @@
 <svelte:head>
 	<title>Collect events from websites, native apps and IoT devices | Your Analytics</title>
   <meta name="description" content="An analytis platform to collect events, such as from website visits, user behavior in native apps or IoT-enabled devices.">
+  {@html '<script async src="https://your-analytics.org/ya.js" data-domain="your-analytics.org"></script>'}
```

Comment:
`@html` can be removed once https://github.com/sveltejs/sapper/pull/1067 is released.